### ISSUE TITLE
[Adaptive Cards> Designer> Card Structure]: Ensures the contrast between foreground and background colors meets WCAG 2 AA minimum contrast ratio thresholds.

### ISSUE THREAD

**Issue by vagpt** [microsoft/AdaptiveCards#8673]
### Target Platforms

Other

### SDK Version

1.6

### Application Name

Adaptive Cards

### Problem Description

**Test environment:**
URL: https://adaptivecards.io/designer/
OS Version: 23H2 (OS Build 25931.1000)
Browser Version:  Edge Dev 118.0.2060.1
Tool: Accessibility Insight For Web

**Repro Steps:**
1. Open the above URL on the EDGE Dev Browser.
2. Run the accessibility insight for web tool and observe the issue.

**Actual Result:**
The luminosity ratio for the 'Creator' text is '3.392.1' which is less than 4.5:1.

**Expected Result:**
Color Contrast ratio for the 'Creator Name' text should be equal to or greater than 4.5:1.

**Element path:** 
.selected.acd-tree-item > .acd-tree-item-text > .acd-tree-item-additionalText

**Snippet:** 
<span class="acd-tree-item-additionalText"> [${creator.name}]</span>

**How to fix:** 
Fix any of the following:
  Element has insufficient color contrast of 3.39 (foreground color: #757575, background color: #bce2ff, font size: 10.5pt (14px), font weight: normal). Expected contrast ratio of 4.5:1

**User Impact:**
It will impact the low vison user as they will not be able to see properly the text if the contrast ratio is less than 4.5:1.

**WCAG Guideline:** 
https://www.w3.org/WAI/WCAG21/Understanding/contrast-minimum.html

"Have feedback to share on Bugs ? Please help fill [Trusted Tester Bug Feedback (office.com)](https://nam06.safelinks.protection.outlook.com/?url=https://forms.office.com/pages/responsepage.aspx?id%3Dv4j5cvGGr0GRqy180BHbR5dUqGQgJ9lItv-jqY8F09lURVdYTENIRkY2UzlCR0NKV1E2VUE5OVpDVy4u&data=04%7c01%7cv-apoorva%40microsoft.com%7c551850210d444be9b28308d9691e3dde%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637656400998658942%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=l6w0SvVcMSvyGnJ6%2B6uSSgd3braF2dZ7/AdBrqqzIR8%3D&reserved=0) “.

### Screenshots

![WCAG1 4 3-Ensures the contrast between foreground and background colors meets WCAG 2 AA minimum contrast ratio thresholds](https://github.com/microsoft/AdaptiveCards/assets/95913748/9392363f-af6e-4a79-87f9-f645fe734239)

![WCAG1 4 3-Ensures the contrast between foreground and background colors meets WCAG 2 AA minimum contrast ratio threshold](https://github.com/microsoft/AdaptiveCards/assets/95913748/adf52905-b462-4700-b0e9-2b5f8a74cdb1)

### Card JSON

```JSON
NA
```

**Comment by vagpt** [comment]
Hi @jwoo-msft ,

As this issue is still repro, and this bug is marked as closed so could you please let us know whether your team will work on this or not also it would be great if please provide us reason for marking this as 'Won't Fix' so that in future we can avoid to logging this type of issue.

**Comment by vagpt** [comment]
Hi @jwoo-msft ,

As this issue is still repro, and this bug is marked as closed so could you please let us know whether your team will work on this or not also it would be great if please provide us reason for marking this as 'Won't Fix' so that in future we can avoide to logging this type of issue.

**Comment by Rishikesh6521** [comment]
Closing this bug and logged a new bug and tracked through #9325
